x-pack/plugins/session_view/common/utils/sort_processes.test.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { sortProcesses } from './sort_processes';
+import { Process } from '../types/process_tree';
+import { mockProcessMap } from '../mocks/constants/session_view_process.mock';
+
+describe('sortProcesses(a, b)', () => {
+  it('works', () => {
```

Comment:
Nit: Could you make the descriptions a bit more descriptive? It's helpful for people who have never seen this code before. Maybe something like `it('sorts processes accurately based on their start times', () => { ... } )`?